src/streamlink/plugins/pluzz.py

```diff
@@ -140,6 +150,13 @@ def _get_streams(self):
             elif '.m3u8' in video_url:
                 for stream in HLSStream.parse_variant_playlist(self.session, video_url).items():
                     yield stream
+            elif '.mp4' in video_url:
+                match = self._mp4_bitrate_re.match(video_url)
+                if match is not None:
+                    bitrate = match.group('bitrate')
+                else:
+                    bitrate = video['format']
+                yield bitrate, HTTPStream(self.session, video_url)
```

Comment:
So for the HTTP streams they'll have a quality name like `mp4-dp`? Is there a way to turn that into something more useful? or replace it with something more useful? `live` is the fallback option, it will get sorted in the list of streams. 
I think `mp4-dp` will not work if it's the only stream available and you try to play `best` (or `worst`). It looks like you'll only ever get one stream when it's `mp4`? Maybe change the name of the stream to `live` as a fallback, or add a `stream_weight` method so they get sorted :)